packages/spec/integration/esbuild/__tests__/production.js

```diff
@@ -0,0 +1,35 @@
+const { handleConsoleOutput } = require('../../../helpers');
+
+describe('esbuild - production', () => {
+  let url, page, getInnerText;
+
+  beforeAll(async () => {
+    const { getUrl } = HopsCLI.start('--production', '--experimental-esbuild');
```

Comment:
Oh, is a production build a valid use-case? Or do we want to disallow this?

Reply:
Good question! Might make sense. But we need documentation of that feature then.

Reply:
Yes, documentation makes sense. Do you want to add it in this PR or create a separate one?
I know that @jhiode also offered to help with esbuild related tasks.

Reply:
I'll do it. Internally as well.

Reply:
I think we should still allow esbuild mode to be used with production, I don't think that we should judge that. The experimental `--fast-build` mode also works with `--production`.

Reply:
Agreed. For the OS Hops this should be fine.